### DISCUSSION TITLE
Any Plans to Release the Application as a Flatpak?

### DISCUSSION THREAD

**Discussion by FeryET** [mockoon/mockoon#1019] [Ideas]
Hello.

I love using Mockoon, but I hate snaps, making a snap package to work is a hell of a task compared to flatpaks, I can easily config, manage and reinstall flatpaks, but I cannot for the life of me make many things on snap packages work (font configurations for example).

Wanted to see if you plan to release a flatpak version of the application. I can contribute to this endeavor if people are interested.

**Comment by 255kb** [comment]
Hey, You are the first to ask and I have no plan to release a flatpak version yet. But I am all in favor for more distribution channels 🙂
I am also not a Linux user, so I am glad if you can contribute.
We are using Electron Builder to package the app and create the binaries. Adding a flatpak format should be easy enough (https://www.electron.build/configuration/flatpak).
The most important thing is to know how to distribute this format, if it needs/can be added to Flat Hub, etc. 
The release distribution is mostly a manual process that I do (publishing to Snap, Microsoft store, etc) and I would need some guidances!

If you want to work on this, could you please create a new issue? I will put it on the roadmap and assign it to you.
